File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/datadriven/DataDrivenViewModelResultRow.cpp
#include "DataDrivenViewModelResultRow.h"
#include <string>

namespace datadriven
{
  std::string DataDrivenViewModelResultRow::getNameLabelText()
  {
    return this->NameLabelText;
  }
  std::string DataDrivenViewModelResultRow::getIdLabelText()
  {
    return this->IdLabelText;
  }
  std::string DataDrivenViewModelResultRow::getDescriptionLabelText()
  {
    return this->DescriptionLabelText;
  }
  std::string DataDrivenViewModelResultRow::getRowHandle()
  {
    return this->RowHandle;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/datadriven/DataDrivenViewModelResultRow.cs
/// <filename>
///     DataDrivenViewModelResultRow.cs
/// </filename>
namespace datadriven
{
    public class DataDrivenViewModelResultRow
    {
        private string NameLabelText;

        public virtual string getNameLabelText()
        {
            return this.NameLabelText;
        }

        private string IdLabelText;

        public virtual string getIdLabelText()
        {
            return this.IdLabelText;
        }

        private string DescriptionLabelText;

        public virtual string getDescriptionLabelText()
        {
            return this.DescriptionLabelText;
        }

        private string RowHandle;

        public virtual string getRowHandle()
        {
            return this.RowHandle;
        }
    }
}
